### DISCUSSION TITLE
Adding an inventory lock feature

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#7101] [Enhancement]
Being able to lock certain items in certain slots of people's inventories. I feel like this would be very useful in a lot of scenarios and should be added to CMI. You would be able to configure this to like permission, config, or command. I would personally use this for a lobby where you will automatically be given a set of items (like a lobby selector and an glide pearl with a certain amount of cooldown) and those get locked in your hotbar.

**Comment by mrfloris** [comment]
This would be kinda nice, but it's a rather big thing as well. There's so many ways to abuse this, that this is better for a third party plugin to deal with like ItemJoin.

**Reply by ghost** [reply]
Yeah that makes sense, but I feel like that would still fit well in CMI's suite and would love to see it added, even if it is a big task. Thanks for the reply
